docs-site/docs/user/project/projects/manage-projects.md

```diff
@@ -75,7 +75,15 @@ To update the URL of the project:
 3. Select **Change project URL**
 
 ## View Project History
-<!-- TODO: Add steps to view project history -->
+
+Prerequisite:
+- You must have at least the Maintainer role for the project.
+
+To view a project's history:
+
+1. Select the project that you want to edit and then on the left sidebar, select **History**
```

Comment:
Please break this into 2 steps like is done on the other page in the PR

Reply:
Fixed in 52b5939cbce0f838bffc5b200d96f444c0d5756d